src/Honeycomb.OpenTelemetry/HoneycombExporter.cs

```diff
@@ -20,75 +19,75 @@ public HoneycombExporter(IHoneycombService honeycombService, IOptions<HoneycombA
             _settings = settings;
         }
 
-        public async override Task<ExportResult> ExportAsync(IEnumerable<SpanData> batch, CancellationToken cancellationToken)
+        public override ExportResult Export(in Batch<Activity> batch)
         {
-            var honeycombEvents = new List<HoneycombEvent>();
-
-            foreach (var span in batch)
+            using (var scope = SuppressInstrumentationScope.Begin())
             {
-                if (span.Attributes
-                        .Any(a => a.Key == "http.url" && 
-                                 (a.Value is string urlStr && 
-                                  urlStr.StartsWith("https://api.honeycomb.io/"))))
-                    continue;
-                
-                var events = GenerateEvent(span);
-                honeycombEvents.AddRange(events);
-            }
-            await _honeycombService.SendBatchAsync(honeycombEvents);
+                try
+                {
+                    var honeycombEvents = new List<HoneycombEvent>();
+                    foreach (var activity in batch)
+                    {
+                        // TODO: skip activities where url starts with https://api.honeycomb.com
```

Comment:
If I remember correctly, the underlying `HoneycombService` already ignores these.

Reply:
Ignore this I've checked and it doesn't, that must have been somewhere else.

Reply:
I left that TODO as I think it was added to ignore http call to send data to honeycomb. Now we have the [SuppressInstrumentationScope](https://github.com/honeycombio/opentelemetry-exporter-dotnet/pull/4/files#diff-7998bad314801105b4e8cec6a0eb2d7a03e6006d56a0436268852ed7b3b4bc8cR24) to prevent auto instrumenting work done within exporters.

If that's the case, we can just remove the TODO.